.github/workflows/android-build.yml

```diff
@@ -40,48 +40,38 @@ jobs:
         with:
           gradle-version: '8.6'
 
-      # Check the NDK that we're using
-      - name: Check Android NDKs
+      # Check the Android SDK and NDK that we're using
+      - name: Check Android SDK and NDK
         run: |
           set -e -x
           uname -m
+          echo "ANDROID_HOME=$ANDROID_HOME"
+          ls -l $ANDROID_HOME
+
+          echo "ANDROID_SDK_ROOT=$ANDROID_SDK_ROOT"
+
           echo "ANDROID_NDK_HOME=$ANDROID_NDK_HOME"
           echo "ANDROID_NDK_LATEST_HOME=$ANDROID_NDK_LATEST_HOME"
           ls -l $ANDROID_HOME/ndk
 
+      - name: Check if emulator is installed and install it if needed
+        run: |
+          if [ -d "${ANDROID_SDK_ROOT}/emulator" ]; then
+              echo "${ANDROID_SDK_ROOT}/emulator exists"
+          else
+              ${ANDROID_SDK_ROOT}/cmdline-tools/latest/bin/sdkmanager --install "emulator"
+          fi
+
       # Needed for linux
       - name: Install jq
         run: |
           sudo apt-get install jq
 
-      - uses: actions/setup-dotnet@v4
-        with:
-          dotnet-version: '8.0.x'
-
       - name: Checkout OnnxRuntime GenAI repo
         uses: actions/checkout@v4
         with:
           submodules: true
 
-      - name: Get the Latest OnnxRuntime Nightly Version
```

Comment:
Removing these steps has no effect on either building or testing GenAI on Android platform.

For building, without explicitly pass `--ort_home`, the version in [ortlib.cmake](https://github.com/microsoft/onnxruntime-genai/blob/main/cmake/ortlib.cmake) will be used.

For testing, the latest release version of ORT is used:

https://github.com/microsoft/onnxruntime-genai/blob/c77e525243dc233b88edaefabbfc5f19facf4044/src/java/src/test/android/app/build.gradle#L44

I remember @baijumeswani  mentions that the nightly version of ORT should be used for testing, but I'm not suer that's available for Android.